sdk/core/azure-core-http-okhttp/pom.xml

```diff
@@ -74,6 +74,13 @@
       <version>1.43.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
     </dependency>
 
+    <!-- transient dependency -->
```

Comment:
please put the tracking issue to remove this in this comment block before committing it